Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:erbium
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN npm install
```

Comment:
I think this (`RUN npm install`) can be removed as it's included as part of the `npm run dev` command.